### PR TITLE
Added Thank you section with Js effect

### LINKED ISSUE #104: Create Thank You For Contributing Section with JS
**Actual Behaviour**

No Thank You Section

**Expected Behaviour**

Should be there
**Steps to reproduce it**

Add it
Add JS effect
Add Multiple Languages

**Would you like to work on the issue?**

Working on it

### PR DESCRIPTION
**Pull request by saarthakchats**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Included a Preview link and screenshot showning after and before the changes.
- [x] Included a description of change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes #104
-  [Preview](https://saarthakchats.github.io/alby/)

## Description / Changes
- Added Thank you for contributing section
- Added typewriter JS effect
- Added in English
- Added in French

### CODE REVIEW

**Comment by liveHarshit** [comment]
Try to add 2-3 more languages. You can add HINDI also.

**Comment by saarthakchats** [comment]
@fragm3 
done!

**Comment by saarthakchats** [comment]
@STEdirisuriya 
done!

**Comment by saarthakchats** [comment]
@liveHarshit @fragm3 
Please merge

File: index.pug [hunk 1]
```diff
@@ -0,0 +1,2 @@
+container
+  #app
```
Comment (simsausaurabh) [inline index.pug]: Add EOF.

**Comment by simsausaurabh** [comment]
@fragm3 Please do not merge when other mentor requested any changes

**Comment by saarthakchats** [comment]
I'll fix these in another PR @simsausaurabh

**Comment by saarthakchats** [comment]
I'll create an issue and do it 💯

**Comment by jogendra** [comment]
@fragm3 Please do not merge without at least two approval from mentors.

**Pull request merged**